### PR TITLE
Feature/implement delete financial transaction

### PR DESCRIPTION
**Pull request by DawidStuzynski**

### CODE REVIEW

**Review by odzio33 — CHANGES_REQUESTED** [review]
changes required:
please create tests for this issues

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java [hunk 1]
```diff
@@ -150,4 +150,14 @@ void shouldSuccessfullyFindFinancialTransaction_WhenFindingWithProperTransaction
         //then
         Assertions.assertEquals(financialTransactionDTO, foundTransaction);
     }
+
+    @Test
```
Comment (odzio33) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java:154]: Please add correct @DisplayName for test.

**Review by odzio33 — CHANGES_REQUESTED** [review]
Please add correct display name for test

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java (hunk 1 above)
Comment (DawidStuzynski) [reply src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java:154]: Okay, done

**Review by dawciobiel — DISMISSED** [review]
Looks good for me

**Review by RobertoJavaDev — DISMISSED** [review]
looks good to me

**Review by odzio33 — DISMISSED** [review]
Looks good for me

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Pull request merged**